x-pack/plugin/logstash/src/main/java/org/elasticsearch/xpack/logstash/util/LogstashPipelinesUtil.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.logstash.util;
+
+/**
+ * An util class to define extra operations of Logstash pipelines.
+ */
+public class LogstashPipelinesUtil {
+
+    public static final String WILDCARD = "*";
+
+    /**
+     * Converts given special characters contained String to regex pattern.
+     * @param input input String which contains special characters.
+     * @return regex pattern based on input string.
+     */
+    public static String toRegex(String input) {
+        StringBuffer stringBuffer = new StringBuffer(input.length());
+        stringBuffer.append('^');
+        for (int i = 0; i < input.length(); i++) {
+            switch(input.charAt(i)) {
+                case '*':
+                    stringBuffer.append(".*");
+                    break;
+                case '?':
```

Comment:
I need to remove special chars since Kibana also does not allow special chars for pipeline name.